Dockerfile

```diff
@@ -1 +1,78 @@
+# Use the base image defined at build time (default to "ubuntu" if not provided)
+ARG BASE_IMAGE=ubuntu
+FROM $BASE_IMAGE
```

Comment:
Please can you specify the tag used in line 3? The tag should should also be a user input.

Reply:
Dockerfile was revised and tag name was added

Reply:
Do you want to modify the base image or the tag? (Line 2)